web/src/app/wizard/WizardForm.js

```diff
@@ -137,7 +137,8 @@ export default class WizardForm extends React.PureComponent {
               required
               type='number'
               placeholder='15'
-              mapOnChangeValue={(value) => value.toString()}
+              min={1}
```

Comment:
I believe this can be 0, which means that it will cause the step to immediately escalate

Reply:
addressed, changes made as requested.